client/tests/Banner.test.js

```diff
@@ -18,4 +19,12 @@ describe('Unit test for <Banner /> Component', () => {
     const wrapper = shallow(<Banner />);
     expect(wrapper.find('button').text()).toEqual('View Photos');
   });
+
+  it('should invoke toggleGalleryView on banner click', () => {
+    const toggleGalleryView = sinon.spy();
```

Comment:
If you would like to limit your dependencies you can also monitor whether a function has been called using a mock function in Jest: https://facebook.github.io/jest/docs/en/mock-functions.html